docs/reference/cli/registries.md

```diff
@@ -20,7 +20,7 @@ List private registries
     SERVER                                        USERNAME
     123456789012.dkr.ecr.us-east-1.amazonaws.com  AKIAABCDE1F2GHIJKLMN
     private.registry.com                          my_private_name
-    index.docker.io/v1/                           my_docker_name
+    https://index.docker.io/v1/                           my_docker_name
```

Comment:
@nightfury1204 could you update your branch with `staging`? This was updated recently.

Reply:
already rebased it with staging

Reply:
no changes